src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -354,7 +354,7 @@ public static bool TryDecodeFrom_Vector128<TChar>(ReadOnlySpan<TChar> source, Sp
                     Vector128<ushort> vec1 = Vector128.LoadUnsafe(ref srcRef, offset).AsUInt16();
                     Vector128<ushort> vec2 = Vector128.LoadUnsafe(ref srcRef, offset + (nuint)Vector128<ushort>.Count).AsUInt16();
 
-                    vec = Ascii.ExtractAsciiVector(vec1, vec2);
+                    vec = Vector128.NarrowNative(vec1, vec2);
```

Comment:
Can you get numbers if you just use `Narrow` instead of creating a new `NarrowNative`?